.github/workflows/ci-build.yaml

```diff
@@ -1,7 +1,7 @@
 name: CI
 
 # We test all supported Python versions as follows:
-# - 3.7  : Documentation build
+# - 3.8  : Documentation build
```

Comment:
Double 3.8?

Reply:
Sorry, I read it wrong

Reply:
Yeah, this list also leaves out 3.11. We should probably rework the test matrix, but I was thinking we'd do that separately from dropping 3.7 to avoid unnecessarily blocking this. What do you think?

Reply:
Sounds good!